### PR TITLE
Add classical training and episodic training notebooks to colab

### PR DESCRIPTION
**Pull request by tnwei**
First of all thanks for creating this repo! I have found it useful, and would like to contribute.

Out of the three example notebooks in the repo, only one directly works on Google Colab. In this PR, I've slightly modified the other two notebooks (`notebooks/episodic_training.ipynb` and `notebooks/classical_training.ipynb`) so that they can also be directly opened in Colab, which should make it even more convenient for first time users to try them out.

### CODE REVIEW

File: notebooks/classical_training.ipynb [hunk 1]
```diff
@@ -431,9 +549,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.10"
+   "version": "3.10.4"
```
Comment (ebennequin) [inline notebooks/classical_training.ipynb:552]: Was the version change needed?

File: notebooks/episodic_training.ipynb [hunk 2]
```diff
@@ -418,6 +516,12 @@
   }
  ],
  "metadata": {
+  "accelerator": "GPU",
+  "colab": {
+   "name": "Copy of episodic_training.ipynb",
```
Comment (ebennequin) [inline notebooks/episodic_training.ipynb:521]: We should remove the "Copy of "

File: notebooks/episodic_training.ipynb [hunk 3]
```diff
@@ -118,6 +166,30 @@
...
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "NfPcFd3wiAp5",
+    "outputId": "ec11a6fe-84bd-4138-977b-b2e9fd1c5692"
+   },
+   "outputs": [],
+   "source": [
+    "# Checks if the CUB dataset is downloaded, triggers download if not\n",
+    "cub_datadir = Path(\"data/CUB/images\")\n",
+    "if (not cub_datadir.exists()) and (not len(list(cub_datadir.glob(\"*\"))) == 200):\n",
```
Comment (ebennequin) [inline notebooks/episodic_training.ipynb:179]: I'm not sure this sanity check is needed here. Same in the other notebook.

File: notebooks/episodic_training.ipynb (hunk 2 above)
Comment (ebennequin) [reply notebooks/episodic_training.ipynb:521]: Same in the other notebook

File: notebooks/episodic_training.ipynb [hunk 4]
```diff
@@ -32,15 +34,53 @@
...
+    "outputId": "26dc469a-3b9f-46eb-950e-ce1d322ad126"
+   },
+   "outputs": [],
+   "source": [
+    "if colab is True:\n",
+    "    # Running in Google Colab\n",
+    "    # Clone the repo\n",
+    "    !git clone https://github.com/sicara/easy-few-shot-learning\n",
+    "    %cd easy-few-shot-learning\n",
+    "    !pip install easyfsl\n",
```
Comment (ebennequin) [inline notebooks/episodic_training.ipynb:65]: We cloned the repo and moved at its root dir, so there is already a package `easyfsl`. Installation from PyPi makes it not obvious what version of `easyfsl` we are using. I think it's better if we don't do the pip install and just use the package inside the repo. (Same in other notebook)

**Review by ebennequin — CHANGES_REQUESTED** [review]
Thank you very much, I agree it would be nice to have these notebooks ready to be used on Colab!

I'm OK with most of the changes, it seems efficient to me. There are some details that I think need fixing, if you're OK with that.

File: notebooks/classical_training.ipynb (hunk 1 above)
Comment (tnwei) [reply notebooks/classical_training.ipynb:552]: Ah, this was auto-generated by Jupyter Lab in the metadata section of the notebook. Yeah we can keep this as-is.

**Comment by tnwei** [comment]
Sorry just got the time to get back to you, just put in the changes requested @ebennequin

**Review by ebennequin — APPROVED** [review]

**Pull request merged**